tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -350,7 +361,7 @@ public function test_update_settings_returns_error_on_non_bool_is_wcpay_enabled_
 		$this->assertEquals( 400, $response->get_status() );
 	}
 
-	public function test_timur_testing() {
```

Comment:
Didn't notice this earlier :D you don't want to have a test in the codebase with your name on it anymore? :D

Reply:
🙈 my bad!